pkg/ccl/changefeedccl/sink_kafka_v2.go

```diff
@@ -164,6 +165,13 @@ func (k *kafkaSinkClientV2) Flush(ctx context.Context, payload SinkPayload) (ret
 				}
 				return nil
 			} else {
+				if len(msgs) == 1 && errors.Is(err, kerr.MessageTooLarge) {
+					msg := msgs[0]
+					err = errors.Wrapf(err,
+						"Kafka message too large: key=%s size=%d mvcc=%s",
+						string(msg.Key), len(msg.Key)+len(msg.Value), msg.Context.Value(mvccTSKey{}).(hlc.Timestamp),
```

Comment:
nit: can you do a nil check on the result of the Value() lookup, just to be a little defensive